fix

OstafiichukO opened the pull request:
передалена разметка, классы, добавлены кнопки

OstafiichukO committed:
fix

OstafiichukO committed:
hover focus

varunyk1805 commented on src/partials/contacts.html:
```diff
@@ -1,85 +1,101 @@
 <section class="section">
   <div class="container contacts">
     <h2 class="section__title section__title--hidden"></h2>
```
потрібно прописати

class="section__title section__title--visually-hidden"

OstafiichukO replied:
готово

varunyk1805 commented on src/partials/contacts.html:
```diff
@@ -1,85 +1,101 @@
     <h2 class="section__title section__title--hidden"></h2>
-
-    <ul class="contacts-list">
+    <ul class="contact-list">
```
поверни на місце букву S

OstafiichukO replied:
Тоді підтягуються стилі яких в моєму css нема, з'являється ще один список на фоні

OstafiichukO replied:
готово

varunyk1805 commented on src/partials/contacts.html:
```diff
@@ -1,85 +1,101 @@
+    </ul>
+    <ul class="contButton-list">
+      <li class="contButton-item">
+        <button type="button" class="btn btn--accent" data-modal-open=""> 
+          <p class="made__more">Our Locations</p> 
+          <span class="btn__vektor btn__vektor--accent">&gt;</span>  
+         </button>
```
пропиши, але забери пробіли після < 

< button type="button" class="btn btn--accent" data-modal-open>Our Locations
  < svg class="btn__icon" width="8" height="8">
    < use href="images/icons.svg#bracket"></use>
  < /svg>
< /button>

OstafiichukO replied:
готово

varunyk1805 commented on src/partials/contacts.html:
```diff
@@ -1,85 +1,101 @@
+         </button>
+    </li>
+      <li class="contButton-item">
+        <button type="button" class="btn" data-modal-open=""> 
+        <p class="btn__text">Franchise</p> 
+        <span class="btn__vektor">&gt;</span>  
+       </button>
```
пропиши, але забери пробіли після <

< button type="button" class="btn" data-modal-open>Our Locations
< svg class="btn__icon" width="8" height="8">
< use href="images/icons.svg#bracket">
< /svg>
< /button>

OstafiichukO replied:
готово

varunyk1805 commented on src/sass/components/_contacts.scss:
```diff
@@ -18,77 +18,139 @@
   }
 }
-.contacts-list {
+.contact-list {
```
переглянь свою розмітку і свої стилі
ти або пишеш усюди contact або contacts

OstafiichukO replied:
відповів вище, contact-list ні на кого не впливає окрім себе

OstafiichukO replied:
готово

varunyk1805 commented on src/partials/contacts.html:
```diff
@@ -1,85 +1,101 @@
+          <span class="btn__vektor btn__vektor--accent">&gt;</span>  
+         </button>
+    </li>
+      <li class="contButton-item">
```
в назвах класів не використовуй заглавні букви, тільки прописні

AnastasiaZhab replied:
в методології БЕМу можна використовувати великі літери для кожного нового слова в назві класу

AnastasiaZhab replied:
в методології БЕМу дозволяється написання кожного нового слова классу з великої літери, це стиль CamelCase

OstafiichukO replied:
готово

OstafiichukO committed:
подправил

Если будет фоновый список, изменить класс contacts-list на contact-list.
Подтягивает стили .contacts-list__item:nth-child(2) {
    padding-bottom: 25px;
}
.contacts-list__item:not(:first-child) {
    padding-top: 25px;
    border-top: 1px solid #e1e1e1;
}

OstafiichukO committed:
подправил классы

OstafiichukO committed:
вакцына
